src/components/common/TitleContentLayout.tsx

```diff
@@ -0,0 +1,29 @@
+interface TitleContentLayoutProps {
```

Comment:
```suggestion
interface TitleContentLayoutProps extends React.PropsWithChildren {
```

React 내장 타입인 PropsWithChildren 를 사용해보는건 어떨까요?
children의 타입을 더 명확하게 표현해주는 장점이 있어요.

Reply:
고민했었는데! 감사합니다앙 😀